starkcore/utils/request/request.go

```diff
@@ -0,0 +1,68 @@
+package request
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/starkbank/ecdsa-go/v2/ellipticcurve/ecdsa"
+	"github.com/starkinfra/core-go/starkcore/environment"
+	Error "github.com/starkinfra/core-go/starkcore/error"
+	"github.com/starkinfra/core-go/starkcore/user/user"
+	"github.com/starkinfra/core-go/starkcore/utils/api"
+	"github.com/starkinfra/core-go/starkcore/utils/checks"
+	urls "github.com/starkinfra/core-go/starkcore/utils/url"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func Fetch(host string, sdkVersion string, user user.User, method string, path string, payload interface{}, apiVersion string, language string, timeout int, query map[string]interface{}) (*http.Response, Error.StarkErrors) {
+	var url string
+	var body string
+	language = checks.CheckLanguage(language)
+
+	if payload != "" {
+		apiJson, _ := json.Marshal(api.CastJsonToApiFormat(payload))
```

Comment:
o cast json to api format não é feito aqui nos outros SDKs

Reply:
Agora sim